lib/OHTAPI.php

```diff
@@ -429,17 +429,17 @@ public function getContext($contextId)
      * @param integer $wordCount (optional)
      * @param string $phraseKeys (optional)
      * @param boolean $retranslate (optional)
-     * @param string $callbackUrl (optional)
+     * @param string $notes
      * @return stdClass response object
      */
-    public function translateContext($contextId, $source, $target, $wordCount = 0, $phraseKeys = '', $retranslate = false, $callbackUrl = '')
+    public function translateContext($contextId, $source, $target, $wordCount = 0, $phraseKeys = '', $retranslate = false, $notes = '')
```

Comment:
@pavlakis thanks for this!
the $callbackUrl parameter is quite important for many customers. 
I suggest adding the $notes as last parameter after the $callbackUrl